### PR TITLE
created avatars of 4 different sizes

### PR DESCRIPTION
**Pull request by vidhi-thakur**

### CODE REVIEW

File: components/component_page/avatar.html [hunk 1]
```diff
@@ -1,12 +1,90 @@
...
\ No newline at end of file
+  </head>
+  <body>
+    <h3>avatars with images</h3>
+    <section class="avatar-box">
+      <div class="avatar avatar-xs-size">
+        <img
+          src="https://avatars.dicebear.com/api/adventurer/vidhi97.svg"
+          alt="avatar"
+        />
```
Comment (nightlight-wander) [inline components/component_page/avatar.html:19]: avatar-xs  (same for others lg ,xlg,s )would be fine ,someone mentioned it in the code review session,like we can undestand from avatar-xs only :),so always try to keep the class names shorter.

**Comment by nightlight-wander** [comment]
Looks good, you can make the first letter capital:),not necessary. You could have done some class names shorter like avatar-xs only for the sizes part ,also for the text you can give a diff class name instead of avatar-xs-text as this class was just used for font size calculation. Similarly for other avatars. Not required though ,someone mentioned in code reviews to keep the class names as short as possible until it can be figured out ,for what purpose they have been used. Overall it looks good.

**Review by nyyshaaa — COMMENTED** [review]
Looks good, you can make the first letter capital:),not necessary. You could have done some class names shorter like avatar-xs only for the sizes part ,also for the text you can give a diff class name instead of avatar-xs-text as this class was just used for font size calculation. Similarly for other avatars. Not required though ,someone mentioned in code reviews to keep the class names as short as possible until it can be figured out ,for what purpose they have been used. Overall it looks good.

**Comment by vidhi-thakur** [comment]
> Looks good, you can make the first letter capital:),not necessary. You could have done some class names shorter like avatar-xs only for the sizes part ,also for the text you can give a diff class name instead of avatar-xs-text as this class was just used for font size calculation. Similarly for other avatars. Not required though ,someone mentioned in code reviews to keep the class names as short as possible until it can be figured out ,for what purpose they have been used. Overall it looks good.

thank you for your review, will update the class names

**Pull request merged**